src/libraries/Microsoft.Extensions.Configuration.Binder/src/ConfigurationBinder.cs

```diff
@@ -698,6 +746,37 @@ private static Array BindArray(Array source, IConfiguration config, BinderOption
                 options);
         }
 
+        // todo: steve - we might not need this; currently, these attributes are only applicable to properties and
+        // not parameters. We might be able to get rid of this method once confirm whether it's needed or not.
+
+        private static string GetParameterName(ParameterInfo parameter)
```

Comment:
It may make sense to refactor this code so it isn't duplicated with GetPropertyName below.

Reply:
Good idea - I was contemplating that, but was in two minds based on the outcome of the decision on whether to apply the attribute on constructor parameters or not.